app/core/Router.coffee

```diff
@@ -183,8 +183,8 @@ module.exports = class CocoRouter extends Backbone.Router
     'play/spectate/:levelID': go('play/SpectateView')
     'play/:map': go('play/CampaignView')
 
-    'premium': go('PremiumFeaturesView')
-    'Premium': go('PremiumFeaturesView')
+    'premium': go('PremiumFeaturesView', { redirectStudents: true, redirectTeachers: true })
+    'Premium': go('PremiumFeaturesView', { redirectStudents: true, redirectTeachers: true })
```

Comment:
Students and teachers should not reach this page via a redirect from the HeroVictoryModal.